### DISCUSSION TITLE
Is it possible to keep a fixed yaw for some turbines in the optimization algorithm while tracking the optimal yaw for other turbines?

### DISCUSSION THREAD

**Discussion by clarissa-eccard** [NatLabRockies/floris#252] [Q&A] [answered]
Hi everyone,

When using the yaw optimization algorithm, I would like to set the yaw for some turbines, ie track the optimal yaw for some turbines while keeping the optimal yaw for others. It is possible?

Thanks

**Comment by bayc** [comment, marked as answer]
Hello,

We are working to integrate this function currently, captured in this pull request: https://github.com/NREL/floris/pull/245

Please see the details of the pull request for how to optimize yaw for only specific turbines while setting the yaw angle for other turbines. This pull request should be merged soon, but in case you need this capability even sooner, you can checkout the pull request and see if it meets your needs.

Cheers!
